userspace/falco/app/actions/init_falco_engine.cpp

```diff
@@ -95,8 +95,8 @@ void configure_output_format(falco::app::state& s) {
 
 	// See https://falco.org/docs/rules/style-guide/
 	const std::string container_info =
-	        "container_id=%container.id container_image=%container.image.repository "
-	        "container_image_tag=%container.image.tag container_name=%container.name";
+	        "container_image=%container.image.repository "
+	        "container_image_tag=%container.image.tag";
```

Comment:
Note for ourselves: we need to update the documentation https://falco.org/docs/rules/style-guide/
cc @LucaGuerra

Reply:
Once this is merged let's open an issue